### ISSUE TITLE
OMP: Error #134: Cannot set thread affinity mask.

### ISSUE THREAD

**Issue by JohnNapier** [pydata/pandas#9328]
This issue has been reported in the past, however there were some problems reproducing it. (https://github.com/pydata/pandas/issues/6720). I'm providing some code that hopefully helps catch the bug (finally!).

The error appears to be related to the division of a large Series or DataFrame that was built by pd.read_hdf(). To see this, please run the following sample code:

```python
import pandas as pd,numpy as np
# Works
df=pd.DataFrame(np.arange(10000).reshape(-1,4),columns=['a','b','c','d'])
df.to_hdf('hdf5.h5','test')
df=pd.read_hdf('hdf5.h5','test')
x=(df['a']-df['b'])/(df['c']-df['d'])
print 'Done for 10000'

# Does not work
df=pd.DataFrame(np.arange(100000).reshape(-1,4),columns=['a','b','c','d'])
df.to_hdf('hdf5.h5','test')
df=pd.read_hdf('hdf5.h5','test')
x=(df['a']-df['b'])/(df['c']-df['d'])
print 'Done for 100000'
```

The first operation is carried out, however the second one produces this error when the division is attempted.

```python
OMP: Error #134: Cannot set thread affinity mask.
OMP: System error #87: The parameter is incorrect.
```

The issue can be avoided by:

1) operating with Numpy arrays, e.g. 

```python
x=(df['a']-df['b']).values/(df['c']-df['d']).values 
```
2) or by setting

```pyhton
pd.computation.expressions.set_use_numexpr(False)
```

3) or by creating the DataFrame from pd.read_csv(). 

I'm running Pandas 0.15.2-1, Numpy 1.8.1-2. Enthought's Canopy 1.5.0.2717.

Thanks

**Comment by jreback** [comment]
@JohnNapier This works for me on windows 7, with the conda distribution (as well as macosx / debian). I recall this is specific to Enthought's packaging AFAICT.

**Comment by JohnNapier** [comment]
Thank you, @jreback. This seems to be related to an older MKL build. Enthought is aware of it and it should be corrected in a future Canopy version.

**Comment by jreback** [comment]
closing as related to the Enthought distro

**Comment by cournape** [comment]
Just to mention this has finally been fixed in Canopy when you update to MKL 11.1.4. I could reproduce the bug w/ MKL 10.3, and could not w/ the 11.1.4

**Comment by jreback** [comment]
fantastic 
thanks @cournape
